torch/distributed/_tensor/ops/_math_ops.py

```diff
@@ -972,30 +976,27 @@ def layer_norm_bwd_strategy(mesh: DeviceMesh, op_schema: OpSchema) -> OpStrategy
             out_placements = map_placements_after_reduction(
                 inp_placements, outer_dims, reduce_dims_map, "sum"
             )
-            output_specs_list.append(
-                DTensorSpec(
-                    mesh=mesh,
-                    placements=out_placements,
-                    tensor_meta=weight_src_spec.tensor_meta,
+            if output_mask[1]:
+                output_specs_list.append(
+                    DTensorSpec(
+                        mesh=mesh,
+                        placements=out_placements,
+                        tensor_meta=weight_src_spec.tensor_meta,
+                    )
                 )
-            )
+            else:
+                output_specs_list.append(None)
         else:
+            assert not output_mask[1]
```

Comment:
`assert not output_mask[1], "output_mask[1] == True while weight argument is not None in native_layer_norm_backward."`

Reply:
Done!